cmd/zenchron-engineering/selfhost.go

```
@@ -106,6 +118,11 @@ func selfhostIssue(rawNumber string, commands commandRunner, stdout io.Writer) e
 	if err != nil {
 		return fmt.Errorf("resolve repository root: %w", err)
 	}
+	runtime, err := resolveGoRuntime(root, commands)
+	if err != nil {
+		return fmt.Errorf("resolve Go runtime before repository mutation: %w", err)
+	}
+	fmt.Fprintf(stdout, "Go runtime: %s\n", runtime)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)